src/rgw/rgw_rest_s3.cc

```diff
@@ -702,6 +702,53 @@ int RGWListBucket_ObjStore_S3::get_params()
   return 0;
 }
 
+int RGWListBucket_ObjStore_S3v2::get_params()
+{
+  //RGWListBucket_ObjStore_S3::get_params();
```

Comment:
Commented out statements like this can be removed

Reply:
Done